docs/source/data_acquisition/UploadData.rst

```diff
@@ -81,6 +81,7 @@ Instantiating the Splunk uploader
 The first step in uploading data is to instantiate an uploader for the location we wish to upload data to.
 For Splunk there are three parameters that need to be passed at this stage, the Splunk host name, a username,
 and a password. You can also pass a parameter for ``port``, by default this value is 8089.
+In addition, The security auth token of ``splunkToken`` can be also passed instead of username and password as same as Splunk QueryProvider.
```

Comment:
Can you change to `splunk_token` (to follow python param naming conventions.
Actually, can it just be `token` ? Since we already know that it's for Splunk. I think `token` is what we used for the data provider.

Reply:
Yes, Of course I can. I did it in the new commit.  The name token has already used as session token in the driver's connect parameter, so I used bearer_token instead of splunkToken referring to Splunk python SDK as following.
https://github.com/splunk/splunk-sdk-python/blob/master/splunklib/binding.py#L523

Thank you for teaching me !